docs/openapi.yaml

```
@@ -219,7 +219,7 @@ components:
         value:
           type: string
           description: |
-            The default value for the input. If this is not set, the user may be prompted to provide a value.
+            The default value for the input. If this is not set, the user may be prompted to provide a value. If a value is set, it will not be configurable and the user will not be able to change it.
```

```suggestion
            The default value for the input. If this is not set, the user may be prompted to provide a value. If a value is set, it should not be configurable by end users.
```

Slightly less prescriptive (UI's can technically do anything)